### PR TITLE
task/KAH-939 (Create Typescript config)

### PR DESCRIPTION
**Pull request by kbuchacz**

### CODE REVIEW

**Review by arispen — CHANGES_REQUESTED** [review]
TSLint has been deprecteded as of 2019:
https://palantir.github.io/tslint/

File: package.json [hunk 1]
```diff
@@ -39,7 +42,23 @@
     "winston": "3.3.3"
   },
   "devDependencies": {
-    "speculate": "1.7.4"
+    "@babel/cli": "^7.8.4",
```
Comment (davvvid1) [inline package.json:45]: There are newer versions of this package. Is there any reason why you put here older one? Also why package.lock.json is not commited?

Comment (kbuchacz) [reply package.json:45]: Package-lock is a good catch! Although I've been basing on most stable solution as I read but I can try to upgrade.

File: .prettierrc.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: "all",
+    singleQuote: false,
+    printWidth: 120,
+    tabWidth: 4
+};
```
Comment (arispen) [inline .prettierrc.js:7]: add newline

**Review by arispen — APPROVED** [review]
one minor

**Review by davvvid1 — APPROVED** [review]

**Pull request merged**